src/panels/config/automation/ha-automation-picker.ts

```diff
@@ -327,14 +328,19 @@ class HaAutomationPicker extends SubscribeMixin(LitElement) {
             const date = new Date(automation.last_triggered);
             const now = new Date();
             const dayDifference = differenceInDays(now, date);
+            const formattedTime = formatShortDateTimeWithConditionalYear(
+              date,
+              this.hass.locale,
+              this.hass.config
+            );
+            const elementId = "last-triggered-" + slugify(automation.entity_id);
             return html`
               ${dayDifference > 3
-                ? formatShortDateTimeWithConditionalYear(
-                    date,
-                    this.hass.locale,
-                    this.hass.config
-                  )
-                : relativeTime(date, locale)}
+                ? formattedTime
+                : html`
+                    <ha-tooltip for=${elementId}> ${formattedTime} </ha-tooltip>
+                    <span .id=${elementId}>${relativeTime(date, locale)}</span>
```

Comment:
We don't need property binding here
```suggestion
                    <span id=${elementId}>${relativeTime(date, locale)}</span>
```

Reply:
> We don't need property binding here

OK, I see that in many places we use with a dot, in other without, like ".for" and "for", perhaps some unification needed...